src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -277,8 +249,8 @@ std::vector<std::function<void()>> RedisStoreClient::TakeRequestsFromSendingQueu
   return send_requests;
 }
 
-void RedisStoreClient::SendRedisCmd(std::vector<std::string> keys,
-                                    std::vector<std::string> args,
+void RedisStoreClient::SendRedisCmd(std::vector<RedisConcurrencyKey> keys,
```

Comment:
I think we can still accept the original `std::vector<std::string> keys`, concurrency key is just `{command.redis_key, key}`

Reply:
In a follow-up PR: we should rename keys to fields which is the term used by redis

Reply:
done, removed keys even.